src/Items/ItemMobHead.h

```diff
@@ -30,34 +32,36 @@ class cItemMobHeadHandler :
 		{
 			return true;
 		}
-		AddFaceDirection(a_BlockX, a_BlockY, a_BlockZ, a_BlockFace);
+		auto placedX = a_BlockX, placedY = a_BlockY, placedZ = a_BlockZ;
+		AddFaceDirection(placedY, placedY, placedZ, a_BlockFace);
 
 		// If the placed head is a wither, try to spawn the wither first:
 		if (a_EquippedItem.m_ItemDamage == E_META_HEAD_WITHER)
 		{
-			if (TrySpawnWitherAround(a_World, a_Player, a_BlockX, a_BlockY, a_BlockZ))
+			if (TrySpawnWitherAround(a_World, a_Player, placedX, placedY, placedZ))
 			{
 				return true;
 			}
 			// Wither not created, proceed with regular head placement
 		}
 
-		return PlaceRegularHead(a_World, a_Player, a_EquippedItem, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace);
+		cItem itemCopy(a_EquippedItem);
```

Comment:
Right you are, I made a copy because I knew I would need it, but then forgot to use it a few lines below.